deployToAzure.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+echo "Initialise terraform for Azure"
+terraform -chdir=terraform init  -backend-config "resource_group_name=$RESOURCE_GROUP_NAME" -backend-config "storage_account_name=tfstate14106" -backend-config "key=terraform.tfstate" -backend-config "container_name=tfstate" -backend-config "access_key=$ACCOUNT_KEY"
```

Comment:
Instead of putting everything in the CLI, you can include these settings in the `backend "azurerm"` section of main.tf.

Reply:
And the access_key can be automatically sourced from ARM_ACCESS_KEY

Reply:
I did have the backend config there, but environment variables were not being accepted in the backend config.